controller/src/main/java/cern/jarrace/controller/Application.java

```diff
@@ -0,0 +1,25 @@
+package cern.jarrace.controller;
```

Comment:
Indeed. Additionally we should have an appropriate header (Copyright + ref to the license) in each file.